src/classrooms/classrooms.controller.ts

```diff
@@ -51,4 +51,19 @@ export class ClassroomsController {
     const result = await this.classroomsService.deleteClassroom(parseInt(id, 10), user.id);
     return result;
   }
+
+  @Put(":id")
+  @Roles(EUserType.TEACHER)
+  async updateClassroom(
+    @Param("id") id: string,
+    @CurrentUser() user: { id: number },
+    @Body() updateClassroomDto: UpdateClassroomDto,
+  ): Promise<ClassroomResponseDto> {
+    const classroom = await this.classroomsService.updateClassroom(
+      parseInt(id, 10),
```

Comment:
I saw this in previous commits as well. Curious as to why we need to do this

Reply:
It was silly mistake from my side, I thought the params are string type. I have corrected it now.